### ISSUE TITLE
Use pdfloc for figuring out highlight annotation position

### ISSUE THREAD

**Issue by rschroll** [rschroll/prsannots#4]
Right now, we're looking for the highlighted text in the page.  But there's no guarantee that we will parse the text into the same string as the reader will, so this is a bit fragile.  Moreover, the entire text of long highlights is not included -- only the first (200?) characters are.  If we can work out what the pdfloc information is telling us, we can get around both of these problems, and probably speed up the creation of these annotations as well.

Here's the [one source of info](http://jerous.thimhallan.org/cms/dokuwiki/doku.php/blog:extract_annotations_from_sony_prs-350) I've found so far.

**Comment by Lidanha** [comment]
Hey,

Did anyone had any progress with figuring out the pdfloc secret?

Thanks,
Lidan

**Comment by peci1** [comment]
@Lidanha Yes, I did :) The docs are at https://github.com/peci1/PDFLocConverter , the Python library is still in alpha version =)

**Comment by Lidanha** [comment]
Wow! Thanks, I will take a look!
